policyengine_canada/parameters/gov/provinces/ab/tax/income/benefits/acfb/base_component/reduction.yaml

```diff
@@ -0,0 +1,22 @@
+description: Alberta reduces its Child and Family Benefit base component at this rate.
+brackets:
+  - threshold:
+      2022-07-01: 0
+    rate:
+      2022-07-01: 0
+  - threshold:
+      2022-07-01: 24_467
+      2023-01-01: 25_935
+    rate:
+      2023-01-01: 0.2011
+
+# Calculated from source
```

Comment:
Can you please leave a comment with the math? Same for the other reduction file.

Reply:
Found the original budgetary source, and had to make a decent amount of adjustments. Also adjusted the folder structure